### ISSUE TITLE
AutoKey does not detect when the keymap is changed via soft-switch

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#34]
```
What steps will reproduce the problem?
1. Use setxkbmap to set multiple keymaps e.g. setxkbmap -layout us,is
2. Start AutoKey - current map works correctly
3. Switch maps using i18n tray icon

Expected: Change to secondary map is detected and AutoKey uses this to read
decode key events.
Actual: AutoKey continues to use the old map.


```

Original issue reported on code.google.com by `cdekter` on 12 Apr 2010 at 7:54

**Comment by GoogleCodeExporter** [comment]
```
GDK provides an event in gtk.gdk.Keymap called "keys-changed" - this will 
probably
fix it in the GTK version. KDE version will remain broken for now.
```

Original comment by `cdekter` on 12 Apr 2010 at 8:14

**Comment by GoogleCodeExporter** [comment]
```
Finally fixed for v0.80.0!
```

Original comment by `cdekter` on 18 Sep 2011 at 11:31
- Changed state: **Fixed**
